frontend/src/stories/__snapshots__/storyshots.test.js.snap

```diff
@@ -23,18 +23,58 @@ exports[`Storyshots Crossing List Header Full Width 1`] = `
   </div>
   <div
     className="CrossingListSortToggle"
+    onClick={[Function]}
```

Comment:
I'm interested in what this `[Function]` does. Is it just a mock function, or a placeholder? or what?

Reply:
The storyshots test renderer is too shallow to actually figure out what's going on with the function, it just knows that there's a function being called on click.